pgpool2-4.6.yaml

```diff
@@ -63,6 +68,59 @@ subpackages:
       runtime:
         - ${{package.name}}
 
+  - name: ${{package.name}}-compat
+    description: "Compatibility package for Upstream pgpool image"
+    dependencies:
+      provides:
+        - pgpool2-compat=${{package.full-version}}
+      runtime:
+        - postgresql
+        - postgresql-client
+        - postgresql-dev
+    pipeline:
+      - name: "Symlink all pgpool binaries to /opt/pgpool-II/bin"
+        runs: |
+          mkdir -p ${{targets.subpkgdir}}/opt/pgpool-II/bin
+          for bin in \
+            pcp_attach_node pcp_detach_node pcp_health_check_stats pcp_node_count \
+            pcp_node_info pcp_pool_status pcp_proc_count pcp_proc_info pcp_promote_node \
+            pcp_recovery_node pcp_stop_pgpool pcp_watchdog_info \
+            pg_enc pg_md5 pgpool pgpool_setup pgproto watchdog_setup wd_cli \
+          ; do
+            ln -s "/usr/bin/${bin}" "${{targets.subpkgdir}}/opt/pgpool-II/bin/${bin}"
+          done
+
+          # Link sample files
+          mkdir -p "${{targets.contextdir}}/opt/pgpool-II/etc"
+          # Create symlinks from /etc/pgpool2/* to /opt/pgpool-II/etc/
+          for bin in \
+            aws_eip_if_cmd.sh.sample follow_primary.sh.sample pool_hba.conf.sample \
+            aws_rtb_if_cmd.sh.sample pcp.conf.sample recovery_1st_stage.sample escalation.sh.sample \
+            pgpool_remote_start.sample replication_mode_recovery_1st_stage.sample failover.sh.sample \
+            pgpool.conf.sample replication_mode_recovery_2nd_stage.sample \
+          ; do
```

Comment:
`for sample in ${{targets.destdir}}/etc/pgpool2/; do...`

Reply:
Updated the PR, thanks